tests/console/btrfs_qgroups.pm

```diff
@@ -80,7 +80,8 @@ sub run {
     # Check limits
     assert_script_run "dd if=/dev/zero bs=10M count=3 of=nofile";
     foreach my $c ('a' .. 'b') {
```

Comment:
btw, is it just me who is confused that `$c` is a or b? :)

Reply:
Absolutely not. It should've been called `$i`!

Reply:
![](https://c.tenor.com/2D5ecEGNJTkAAAAd/laughing-christian-bale.gif)